CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # sidekiq_publisher
 
+## 4.1.0
```

Comment:
I missed responding in Slack, but I think I'd favor a possible breaking change being a major bump.